### PR TITLE
Display all storage units in the employees' page

### PR DESCRIPTION
**Pull request by Tajeu2001**

### CODE REVIEW

**Review by layersony — APPROVED** [review]
Next time gitignore the virtual environment or you can delete the one you have and create a another one with a different  name then gitignore it

File: employee/views.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
-from django.shortcuts import render
+from django.shortcuts import render 
 from django.http import HttpResponse
 
 def index(request):
```
Comment (peterken674) [inline employee/views.py:4]: Please delete this view if you are no longer using it.

**Review by peterken674 — CHANGES_REQUESTED** [review]
Kindly add virtual/ to .gitignore. It is not supposed to be pushed to GitHub. Also please look into the other comment inside the view.

**Review by peterken674 — APPROVED** [review]
Good.

**Review by AbdiazizH — APPROVED** [review]
everything is up to date

**Pull request merged**